### PR TITLE
Area Detector

### PR DESCRIPTION
**Pull request by elParcero**
Using file handler to get file size and list (EigerHandler and AreaDetectorTiffHandler).

### CODE REVIEW

File: exercises/0002_eiger_handler.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                                        unique_resources.add(resource_id)
+                                    datum_gen = db.reg.datum_gen_given_resource(resource_id)
+                                    datum_kwargs = [datum['datum_kwargs'] for datum in datum_gen]
+                                    timestamp = time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(int(event['time'])))
+                                    timestamp = time.strptime(timestamp, '%Y-%m-%d %H:%M:%S')
+                                    timestamp = datetime.datetime.fromtimestamp(mktime(timestamp))
+                                    # get the file handler using this
+                                    fh = db.reg.get_spec_handler(resource_id)
+                                    print(fh)
+                                    file_sizes = sum(fh.get_file_sizes(datum_kwargs))
```
Comment (jrmlhermitte) [inline exercises/0002_eiger_handler.py]: call `get_file_list` here, get the file list and then get the file size from there.

Comment (elParcero) [reply exercises/0002_eiger_handler.py]: The file list that is returned includes more than one file, which means I would have to get the size for both files, sum them up, and then assign the sum of the file usage to the file size correct? So I'd need a loop in there?

Comment (jrmlhermitte) [reply exercises/0002_eiger_handler.py]: yep that's right. you can use list comprehension:
```py
filenames = fh.get_file_list(...)
total_size = sum([os.path.getsize(fliename) for filename in filenames])

```

Comment (elParcero) [reply exercises/0002_eiger_handler.py]: I think I got it, I'll push it right now

Comment (elParcero) [reply exercises/0002_eiger_handler.py]: Oh wow, I didn't see your message, but that's exactly what I did, thanks!

Comment (jrmlhermitte) [reply exercises/0002_eiger_handler.py]: awesome :-)

**Comment by jrmlhermitte** [comment]
this looks good, we can merge this

**Pull request merged**